app/controllers/v0/notifications_controller.rb

```diff
@@ -7,6 +7,7 @@ class NotificationsController < ApplicationController
 
     before_action -> { validate_subject!(subject) }
     before_action :set_account
+    before_action :set_notification, only: :show
```

Comment:
Do we need a before action if it's only used for `:show`? I could see this if `set_notification` was in a a base controller class. There could be a base because `set_notification` and `set_account` exists in `DismissedStatusesController` as well.

Reply:
Yeah, it is actually also going to be used in the `Notification#update` endpoint that I'm shipping next.